modules/ssh/agent_test.go

```diff
@@ -0,0 +1,42 @@
+package ssh
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestSshAgentWithKeyPair(t *testing.T) {
+	keyPair := GenerateRSAKeyPair(t, 2048)
+	sshAgent := SshAgentWithKeyPair(t, keyPair)
+	defer sshAgent.Stop()
+
+	// ensure that socket directory is set in environment, and it exists
+	sockFile := filepath.Join(sshAgent.socketDir, "ssh_auth.sock")
+	sockFileEnv, found := os.LookupEnv("SSH_AUTH_SOCK")
+	assert.FileExists(t, sockFile)
+	assert.True(t, found)
+	assert.Equal(t, sockFileEnv, sockFile)
+
+	// assert that there's 1 key in the agent
+	keys, _ := sshAgent.agent.List()
```

Comment:
Don't ignore errors. `require.NoError` on it instead.

Reply:
Fixed